SimPEG/regularization/sparse.py

```diff
@@ -171,7 +171,7 @@ class SparseDeriv(BaseSparse):
     """
 
     def __init__(self, mesh, orientation='x', **kwargs):
-
+        self._length_scales = None
```

Comment:
should we make this a property? That way if we serialize the class, they are included? 
```length_scales = properties.Array("length scales")```